libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -489,6 +493,8 @@ bool AP_AHRS_NavEKF::get_position(struct Location &loc) const
     }
 #endif
     }
+
+    // fall back to position from DCM
     return AP_AHRS_DCM::get_position(loc);
```

Comment:
```suggestion
   if (!always_use_EKF()) {
        return AP_AHRS_DCM::get_position(loc);
    }
   return false;
```

I think this is the entire equivalent behavior patch set.